benchmarks/cloudsuite3/inmem-analytics/run.sh

```diff
@@ -136,25 +132,27 @@ done
 rm -rf results
 mkdir -p results
 
-run_setup
+docker create --name movielens-data cloudsuite/movielens-dataset
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.